docs/account-management/plans.md

```diff
@@ -35,9 +35,9 @@ out to
 
 In Semaphore 2.0, all organizations are on monthly plans.
 
-**Note:** If you are building open source projects on a Paid plan, those builds 
-will be charged accourding to our [pricing](https://semaphoreci.com/pricing).
-Open source projects can be built free of charge only on an [Open Source
+**Note:** If you are building open source projects on a Paid plan, you will be 
+charged accourding to our [pricing page](https://semaphoreci.com/pricing) for those
+builds, too. Open source projects can be built free of charge only on an [Open Source
```

Comment:
those is more like 'oni bildovi' and we need 'za te bildove' so I would change 'those builds' to 'these builds'.